admin/services/check-window.service.js

```diff
@@ -2,13 +2,31 @@
 const CheckWindow = require('../models/check-window')
 
 const checkWindowService = {
-  getCurrentCheckWindow: async function () {
+
+  /**
+   * Get check window for the current time.
+   * @returns {Promise.<*>}
+   */
+  getCurrentCheckWindow: async () => {
     const now = new Date()
     const checkWindow = await CheckWindow.findOne({startDate: {$lte: now}, endDate: {$gte: now}}).exec()
     if (!checkWindow) {
       throw new Error('No checkwindow is currently available')
     }
     return checkWindow
+  },
+  /**
+   * @param url
+   * @returns {*}
+   */
+  addForwardSlashToUrl: (url) => {
```

Comment:
Seems out of place as a public method.

Reply:
In the end I'm not using it, it'll be removed.